src/Equinor.ProCoSys.IPO.Command/InvitationCommands/CreateInvitation/CreateInvitationCommandValidator.cs

```diff
@@ -38,21 +38,17 @@ public CreateInvitationCommandValidator(IInvitationValidator invitationValidator
                 .WithMessage(command =>
                     $"Title must be between {Invitation.TitleMinLength} and {Invitation.TitleMaxLength} characters! Title={command.Title}")
                 //business validators
-                .Must(command => MustHaveValidScope(command.McPkgScope, command.CommPkgScope))
-                .WithMessage(command =>
-                    "Not a valid scope! Choose either mc scope or comm pkg scope")
+                .Must(command => MustHaveValidScope(command.Type, command.McPkgScope, command.CommPkgScope))
+                .WithMessage("Not a valid scope! Choose either DP with mc scope or MDP with comm pkg scope")
                 .Must(command => TwoFirstParticipantsMustBeSetWithCorrectOrganization(command.Participants))
-                .WithMessage(command =>
-                    "Contractor and Construction Company must be invited!")
+                .WithMessage("Contractor and Construction Company must be invited!")
                 .Must(command => RequiredParticipantsHaveLowestSortKeys(command.Participants))
-                .WithMessage(command =>
-                    "SortKey 0 is reserved for Contractor, and SortKey 1 is reserved for Construction Company!")
+                .WithMessage("SortKey 0 is reserved for Contractor, and SortKey 1 is reserved for Construction Company!")
```

Comment:
Maybe change to a more "human" message? "Contractor must be first and Construction Company must be second!